ui/app/src/components/ToolsPanelEmbeddedAppViewButton/ToolsPanelEmbeddedAppViewButton.tsx

```diff
@@ -31,7 +31,7 @@ const useStyles = makeStyles((theme) =>
     },
     toolPanelBody: () => {
       const toolbarMixin: any = theme.mixins.toolbar;
-      const key1 = '@media (min-width:0) and (orientation: landscape)';
+      const key1 = '@media (min-width:0px) and (orientation: landscape)';
```

Comment:
Was the `0` without px causing an issue?
A min-width of zero doesn't make a great deal of sense anyway. Wouldn't this work with just the orientation?